plugin.json

```diff
@@ -1,15 +1,17 @@
 {
     "id" : "multisheet-excel-export",
-    "version" : "1.1.2",
+    "version" : "1.1.3",
 
 
     "meta" : {
         "label" : "Multisheet Excel Export",
+        "category": "Excel",
         "description" : "Export multiple datasets to one xlsx file with multiple sheets",
         "author" : "Dataiku (Kasim PATEL, Thibault DESFONTAINES)",
         "icon" : "icon-file",
         "licenseInfo" : "Apache Software License",
         "url" : "https://www.dataiku.com/product/plugins/multisheet-excel-export",
-        "tags" : ["Productivity"]
+        "tags" : ["Productivity"],
```

Comment:
The [dss-plugin-excel-templater](https://github.com/dataiku/dss-plugin-excel-templater/tree/main) currently has a "Excel" tag [here](https://github.com/dataiku/dss-plugin-excel-templater/blob/bf7dc1e16f134c03065bc8d0289b441a7cb7234c/plugin.json#L13). I don't know if we want the same ?

Reply:
Good point, the tag is used in the Plugin store
Done in ae1ece79f986a6bf8cf45ecc712edd070bc33b0f